scripts/release/mule/sign/linux/amd64/sign.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
add -pipefail, since you're piping error codes in 
```git archive --prefix="algorand-$FULLVERSION/" "$BRANCH" | gzip >| "$PKG_DIR/algorand_${CHANNEL}_source_${VERSION}.tar.gz"```

btw - could you try and spellcheck this one ?

Reply:
I'll add pipefail.  I have shellcheck integrated with vim, so it will always check on saving/closing the file.

It didn't flag the pipeline. Is there anything in particular that caught your attention?

Reply:
Yes, if the git command returns a non-zero exit code with no file names, the gzip would not fail, and set the error code to zero. that's not what we want, right ?

Reply:
Yes, that makes sense and I've added the pipefail option.  I was curious about why you were asking for the shellcheck; is it because of the `>|` redirect?